content/en/getting_started/tagging/unified_service_tagging.md

```diff
@@ -464,19 +464,21 @@ class CdkStack extends cdk.Stack {
 }
 ```
 
+
 [1]: https://docs.aws.amazon.com/cdk/latest/guide/tagging.html
 [2]: https://docs.datadoghq.com/serverless/serverless_integrations/macro
-
 {{% /tab %}}
 
 {{% tab "Custom" %}}
 
 Apply the `env`, `service` and `version` tags following the AWS instructions for [Tagging Lambda Functions][1].
 
-[1]: https://docs.aws.amazon.com/lambda/latest/dg/configuration-tags.html
 
+[1]: https://docs.aws.amazon.com/lambda/latest/dg/configuration-tags.html
 {{% /tab %}}
 
+Please ensure the `DdFetchLambdaTags` option is set to `true` on the CloudFormation stack for your [Datadog Forwarder][14]. This option defaults to `true` since version `3.19.0`.
```

Comment:
```suggestion
Ensure the `DdFetchLambdaTags` option is set to `true` on the CloudFormation stack for your [Datadog Forwarder][14]. This option defaults to `true` since version `3.19.0`.
```